src/components/Modal/BaseModal.tsx

```diff
@@ -181,7 +181,7 @@ function BaseModal(
             onModalHide={hideModal}
             onModalWillShow={() => ComposerFocusManager.resetReadyToFocus()}
             onDismiss={handleDismissModal}
-            onSwipeComplete={onClose}
+            onSwipeComplete={() => onClose()}
```

Comment:
Nether default value `() => {}` is defined nor null check is handled
```suggestion
            onSwipeComplete={() => onClose?.()}
```

Reply:
Updated!